src/style.css

```diff
@@ -1 +1,39 @@
-/* add styles here */
+body {
```

Comment:
According to the checklist, each block should have its own CSS file. Here, all the styles are placed in the single file.